spring-cloud-generator/generate-all.sh

```diff
@@ -18,6 +20,7 @@ save_error_info () {
 # repos are downloaded once before all generation jobs and then removed
 
 bash download-repos.sh
+touch run-sanity-check/generate-all-started
 libraries=$(cat $WORKING_DIR/library_list.txt | tail -n+2)
 while IFS=, read -r library_name googleapis_location coordinates_version googleapis_commitish; do
   echo "processing library $library_name"
```

Comment:
Maybe also write this to a file so that you can later use it in the sanity check?
Can we also tell for each library if there were any changes and log that?